### PR TITLE
outdent err block

### PR DESCRIPTION
**Pull request by zouyee**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE

```

### CODE REVIEW

File: pkg/volume/util.go [hunk 1]
```diff
@@ -274,9 +274,8 @@ func CalculateTimeoutForVolume(minimumTimeout, timeoutIncrement int, pv *v1.Pers
 	timeout := (pvSize / giSize) * int64(timeoutIncrement)
 	if timeout < int64(minimumTimeout) {
 		return int64(minimumTimeout)
-	} else {
-		return timeout
-	}
+	} else
```
Comment (jhorwit2) [inline pkg/volume/util.go]: I think you forgot an `else` here.

**Review by jhorwit2 — CHANGES_REQUESTED** [review]
Thanks for the PR @zouyee! I think you have a syntax error but overall LGTM after that!

**Comment by zouyee** [comment]
ping @jhorwit2

File: pkg/volume/util.go [hunk 2]
```diff
@@ -304,7 +304,7 @@ func GenerateVolumeName(clusterName, pvName string, maxLength int) string {
 	return prefix + "-" + pvName
 }
 
-// Check if the path from the mounter is empty.
+// GetPath Check if the path from the mounter is empty.
```
Comment (davidz627) [inline pkg/volume/util.go]: nit: GetPath {checks}

**Comment by davidz627** [comment]
please run hack/update-gofmt.sh on your changed files. I think it is failing the gofmt verify script.

**Comment by zouyee** [comment]
ping @davidz627

File: pkg/volume/util.go [hunk 3]
```diff
@@ -304,7 +304,7 @@ func GenerateVolumeName(clusterName, pvName string, maxLength int) string {
 	return prefix + "-" + pvName
 }
 
-// Check if the path from the mounter is empty.
+// GetPath Checks if the path from the mounter is empty.
```
Comment (davidz627) [inline pkg/volume/util.go]: nit: lowercase *checks*

File: pkg/volume/util.go [hunk 4]
```diff
@@ -274,9 +274,9 @@ func CalculateTimeoutForVolume(minimumTimeout, timeoutIncrement int, pv *v1.Pers
 	timeout := (pvSize / giSize) * int64(timeoutIncrement)
 	if timeout < int64(minimumTimeout) {
 		return int64(minimumTimeout)
-	} else {
-		return timeout
 	}
+
```
Comment (davidz627) [inline pkg/volume/util.go]: nit: remove space

File: pkg/volume/util.go [hunk 5]
```diff
@@ -87,9 +87,9 @@ func internalRecycleVolumeByWatchingPodUntilCompletion(pvName string, pod *v1.Po
 			// Recycler will try again and the old pod will be hopefuly deleted
 			// at that time.
 			return fmt.Errorf("old recycler pod found, will retry later")
-		} else {
-			return fmt.Errorf("unexpected error creating recycler pod:  %+v\n", err)
 		}
+		return fmt.Errorf("unexpected error creating recycler pod:  %+v", err)
+
```
Comment (davidz627) [inline pkg/volume/util.go]: nit: remove space.

Comment (zouyee) [reply pkg/volume/util.go]: It has been fixed, thanks again.

**Comment by zouyee** [comment]
ping @gnufied

**Pull request merged**